src/main/resources/schemas/schema-form.json

```diff
@@ -33,6 +33,11 @@
                         "x-schema-form": {
                             "expression-language": true
                         }
+                    },
+                    "appendToExistingArray": {
+                        "title": "Append the value to existing queryParam?",
```

Comment:
maybe we should be indicated that this is in the case of a table with the format ?a=1&a=2  ? WDYT ?